### PR TITLE
Improve channel_bind code coverage

### PR DESCRIPTION
**Pull request by songjiayang**
1. change ChannelBind's ID to turn.ChannelNumber.
2. move channelBindTimeout to StartArguments.

### CODE REVIEW

File: internal/allocation/channel_bind.go [hunk 1]
```diff
@@ -4,15 +4,17 @@ import (
 	"fmt"
 	"net"
 	"time"
+
+	"github.com/gortc/turn"
 )
 
-const channelBindTimeout = time.Duration(10) * time.Minute
+var channelBindTimeout = time.Duration(10) * time.Minute
```
Comment (Sean-Der) [inline internal/allocation/channel_bind.go]: Can we make this an argument to the server instead?

That way users can configure via https://github.com/pion/turn/blob/master/turn.go#L23 and then we can keep the default `10` minutes still!

Also would be great to have tests for this. I think we are/were leaking at one point, and we can close the ticket if you can assert that timeouts actually work/tear down everything :)

**Comment by Sean-Der** [comment]
Fantastic improvement! Nice work :) 

Also if we make sure we are properly tearing down channels I think we can close out https://github.com/pion/turn/issues/27

File: internal/allocation/channel_bind.go (hunk 1 above)
Comment (songjiayang) [reply internal/allocation/channel_bind.go]: Put `channelBindTimeout ` into start configure params is a good idea, I will have a try.

**Comment by songjiayang** [comment]
Hi @Sean-Der , review again please.

 I just add a attribute named `permissionLifetime `  to server. I think `ChannelBind` should have the same lifetime as `CreatePermission` lifetime, so it's 5 minutes for default.

File: internal/server/server.go [hunk 2]
```diff
@@ -42,6 +43,7 @@ func NewServer(realm string, a AuthHandler) *Server {
 		authHandler:        a,
 		manager:            &allocation.Manager{},
 		reservationManager: &allocation.ReservationManager{},
+		permissionLifetime: 5 * time.Minute,
```
Comment (Sean-Der) [inline internal/server/server.go]: Can we keep the value before `10 minutes`

beyond that LGTM! If you can change that will approve, thanks @songjiayang :)

Comment (songjiayang) [reply internal/server/server.go]: @Sean-Der now move channelBindTimeout to StartArguments, and change it to 10 minutes with default.

**Comment by songjiayang** [comment]
LGTM

**Pull request merged**